dialogue-apache-hc5-client/src/test/java/com/palantir/dialogue/hc5/CleanerSupportTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue.hc5;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assumptions.assumeThat;
+
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.Test;
+
+class CleanerSupportTest {
+
+    @Test
+    void testEnabled() {
+        boolean isJava8 = "1.8".equals(System.getProperty("java.specification.version"));
```

Comment:
Junit exposes `EnabledForJreRange` to manage which JREs test should run on

Reply:
We're both testing that it's enabled on java9+ and that it's disabled on 11. I think we can use EnabledForJreRange for `testCleaner` though.